src/Controller/PermutSearchController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Controller;
+
+use App\Repository\UserRepository;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class PermutSearchController extends AbstractController
+{
+    /**
+     * @Route("permutsearch", name="permutsearch")
```

Comment:
honest question : does this work without the first "/" ?

Reply:
yes it does